api/lambdas/sparql.ts

```diff
@@ -0,0 +1,99 @@
+import got from 'got';
+import type {
+  APIGatewayProxyEventV2,
+  APIGatewayProxyResult,
+} from 'aws-lambda';
+import { URL } from 'url';
+import { pick } from 'lodash';
+
+export const getStringFromEnv = (key: string): string | Error =>
```

Comment:
This would be a great function to add to a `utils.ts` file ;-).

Reply:
On a serious note: is it worth throwing the error here if we don't have the value vs returning Error? It would save you from having to do the `instanceof` check later on.

Reply:
Moved to `env-utils.ts` in 70221d8.

Reply:
I need to do _something_ to figure out if I got the value back or not. I'm either going to check the return value, or I'm going to have to wrap the call to `getStringFromEnv` in a `try / catch`. It feels cleaner to just return a value in the failure case rather than take an alternative path back out of the function (`try / catch`).